src/api/config/environments/development.rb

```diff
@@ -11,6 +11,18 @@
   # Do not eager load code on boot.
   config.eager_load = false
 
+  # Eager load STI variants, otherwise Rails do not load the STI subclasses in
+  # development environment and we face "uninitalized constant Something" errors.
+  sti_classes_to_eager_load = Dir["app/models/bs_request/**/*.rb",
+                                  "app/models/event/**.rb",
+                                  "app/models/history_element/**.rb",
+                                  "app/models/owner_search/**.rb",
+                                  "app/models/token/**.rb"]
```

Comment:
Loading the token errors breaks for me now in development. Maybe because the `token/errors.rb` contains subclasses?

Reply:
Breaks how?

Reply:
right when I spin up the server it throws ```frontend_1  | 08:42:04 web.1     | Exiting
frontend_1  | 08:42:04 web.1     | /obs/src/api/app/models/token.rb:18:in `<class:Token>': uninitialized constant Token::Errors (NameError)
frontend_1  | 08:42:04 web.1     | 
frontend_1  | 08:42:04 web.1     |   include Token::Errors
frontend_1  | 08:42:04 web.1     |                ^^^^^^^^
frontend_1  | 08:42:04 web.1     |      from /obs/src/api/app/models/token.rb:1:in `<top (required)>'
frontend_1  | 08:42:04 web.1     |      from /usr/lib64/ruby/gems/3.1.0/gems/zeitwerk-2.6.0/lib/zeitwerk/kernel.rb:27:in `require'
frontend_1  | 08:42:04 web.1     |      from /usr/lib64/ruby/gems/3.1.0/gems/zeitwerk-2.6.0/lib/zeitwerk/kernel.rb:27:in `require'
frontend_1  | 08:42:04 web.1     |      from /obs/src/api/app/models/token/errors.rb:1:in `<top (required)>'
frontend_1  | 08:42:04 web.1     |      from /usr/lib64/ruby/gems/3.1.0/gems/zeitwerk-2.6.0/lib/zeitwerk/kernel.rb:35:in `require'
frontend_1  | 08:42:04 web.1     |      from /usr/lib64/ruby/gems/3.1.0/gems/zeitwerk-2.6.0/lib/zeitwerk/kernel.rb:35:in `require'
frontend_1  | 08:42:04 web.1     |      from /usr/lib64/ruby/gems/3.1.0/gems/activesupport-6.1.5.1/lib/active_support/dependencies/zeitwerk_integration.rb:51:in `require_dependency'
frontend_1  | 08:42:04 web.1     |      from /obs/src/api/config/environments/development.rb:23:in `block (3 levels) in <top (required)>'
frontend_1  | 08:42:04 web.1     |      from /obs/src/api/config/environments/development.rb:23:in `each'
frontend_1  | 08:42:04 web.1     |      from /obs/src/api/config/environments/development.rb:23:in `block (2 levels) in <top (required)>'
```

Reply:
See #12892